File: frontend/src/constants/answer.ts
import { Answer, AnswerType } from '@/types/answer';

export const ANSWER_PHRASE: Record<AnswerType | string, string> = {
  BAD: 'X',
  GOOD: 'O',
  NONE: '-',
};

export const ANSWER_OPTIONS: Answer[] = [
  { id: 1, name: 'GOOD' },
  { id: 2, name: 'BAD' },
];

File: backend/bang-ggood/src/main/java/com/bang_ggood/question/domain/Answer.java
package com.bang_ggood.question.domain;

import com.bang_ggood.global.exception.BangggoodException;
import com.bang_ggood.global.exception.ExceptionCode;
import java.util.Arrays;

public enum Answer {

    GOOD, BAD, NONE;

    public static Answer from(String grade) {
        return Arrays.stream(Answer.values())
                .filter(value -> value.name().equals(grade))
                .findFirst()
                .orElseThrow(() -> new BangggoodException(ExceptionCode.ANSWER_INVALID));
    }
}
